Source/WarpX.cpp

```diff
@@ -1491,10 +1491,9 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
     }
 #endif
 
-    bool deposit_charge = do_dive_cleaning || (plot_rho && do_back_transformed_diagnostics);
+    bool deposit_charge = do_dive_cleaning;
```

Comment:
If plot_rho is not used anymore, we can also remove its definition from WarpX.H
and wherever setplot_rho() is defined and/or called

Reply:
Good point, thanks Reva! I will check if we can remove `plot_rho` and associated functions from the code.